src/uk/gov/defra/ffc/Provision.groovy

```diff
@@ -37,8 +37,6 @@ class Provision implements Serializable {
 
       ctx.withEnv(getMigrationEnvVars(ctx, environment, repoName, pr)) {
         ctx.dir(migrationFolder) {
-          // migrations may change in different builds of a PR, refresh schema to avoid errors
-          ctx.sh("docker-compose -p $repoName-$pr -f docker-compose.migrate.yaml run schema-down")
```

Comment:
Turns out we were running schema down twice, as it's done earlier in 'deletePrResources. I've removed the duplicate call here.